test/language/statements/variable/arguments-strict-list-final-init.js

```diff
@@ -3,13 +3,16 @@
 
 /*---
 es5id: 12.2.1-28-s
+esid: sec-variable-statement
 description: >
     arguments as local var identifier assigned to throws SyntaxError
     in strict mode
 flags: [onlyStrict]
+negative:
+  phase: parse
+  type: SyntaxError
 ---*/
 
+throw "Test262: This statement should not be evaluated.";
 
-assert.throws(SyntaxError, function() {
-    eval('function foo() { var a, arguments = 42;}');
-});
+var a, arguments = 42;
```

Comment:
these changes are great, but in this case I'd keep a test with the function declarations. I wonder if we could double these as a test file with and another without the function. WDYT?

Reply:
@jugglinmike ping ^^